charts/actions-runner-controller/templates/deployment.yaml

```
@@ -122,6 +122,9 @@ spec:
         - name: {{ $key }}
           value: {{ $val | quote }}
         {{- end }}
+        { { - if .Values.additionalFullEnv } }
+          { { - toYaml .Values.additionalFullEnv | nindent 8 } }
+        { { - end } }
```

Thanks for your effort! One thing- Probably we'd better guide existing users to use this over the legacy style, without requiring the key change.

So, could you try something like:

```suggestion
        {{- if kindIs "slice" .Values.env }}
        {{- toYaml .Values.env | nindent 8 }}
        {{- else }}
        {{- range $key, $val := .Values.env }}
        - name: {{ $key }}
          value: {{ $val | quote }}
        {{- end }}
        {{- end }}
```

Also, I think adding a space between `{`s would never work(AFAIK).
Have you actually tested this? (Sorry if you did!
Try running `helm upgrade --install $RELEASE_NAME ./charts/actions-runner-controller` locally so that you can test the updated version of the chart before submitting a PR.

---

I did, but ofc forgot to push the fix... Fridays :)